docs/self-managed/platform-deployment/helm-kubernetes/guides/ingress-setup.md

```diff
@@ -75,11 +75,13 @@ webModeler:
 console:
   contextPath: "/console"
 
-zeebe-gateway:
+zeebeGateway:
+  contextPath: "/zeebe"
```

Comment:
:thought_balloon: Same as above, only really need the gRPC ingress since the global ingress can route to the REST API via the context path.